datalad/distribution/create_sibling_github.py

```diff
@@ -80,41 +133,9 @@ class CreateSiblingGithub(Interface):
     with that specific login name.
     """
 
-    _params_ = dict(
-        dataset=Parameter(
-            args=("--dataset", "-d",),
-            doc="""specify the dataset to create the publication target for. If
-                no dataset is given, an attempt is made to identify the dataset
-                based on the current working directory""",
-            constraints=EnsureDataset() | EnsureNone()),
-        reponame=Parameter(
-            args=('reponame',),
-            metavar='REPONAME',
-            doc="""GitHub repository name. When operating recursively,
-            a suffix will be appended to this name for each subdataset""",
-            constraints=EnsureStr()),
-        recursive=recursion_flag,
-        recursion_limit=recursion_limit,
-        name=Parameter(
-            args=('-s', '--name',),
-            metavar='NAME',
-            doc="""name to represent the GitHub repository in the local
-            dataset installation""",
-            constraints=EnsureStr()),
-        existing=Parameter(
-            args=("--existing",),
-            constraints=EnsureChoice('skip', 'error', 'reconfigure', 'replace'),
-            metavar='MODE',
-            doc="""desired behavior when already existing or configured
-            siblings are discovered. In this case, a dataset can be skipped
-            ('skip'), the sibling configuration be updated ('reconfigure'),
-            or process interrupts with error ('error'). DANGER ZONE: If 'replace'
-            is used, an existing github repository will be irreversibly removed,
-            re-initialized, and the sibling (re-)configured (thus implies 'reconfigure').
-            `replace` could lead to data loss, so use with care.  To minimize
-            possibility of data loss, in interactive mode DataLad will ask for
-            confirmation, but it would raise an exception in non-interactive mode.
-            """,),
+    _params_ = _GitHub.create_sibling_params
+    # deprecated option
+    _params_.update(
```

Comment:
all of them (since above `_params_`) or just `--github-login` (move one line down)? I think `--github-organization` and others remains valid. If so:
```suggestion
    _params_.update(
        # deprecated option
```
but even then -- not sure if to be deprecated, per concern stated in another comment -- how to choose between multiple credentials, where knowing the login would help to disambiguate

Reply:
It refers to all of them.